.github/workflows/aarch64.yml

```
@@ -45,7 +45,7 @@ jobs:
           args: make std_spec
   aarch64-musl-test-compiler:
     needs: aarch64-musl-build
-    runs-on: [runs-on, runner=2cpu-linux-arm64, "family=m7g", ram=8, "run-id=${{ github.run_id }}"]
+    runs-on: [runs-on, runner=2cpu-linux-arm64, "family=m7g", ram=16, "run-id=${{ github.run_id }}"]
```

```suggestion
    runs-on: [runs-on, runner=4cpu-linux-arm64, "family=m7g", ram=16, "run-id=${{ github.run_id }}"]
```
In the past you had to also bump cpu count as i think it goes lock step with memory. Would explain why the job seems to be stuck waiting: https://github.com/crystal-lang/crystal/actions/runs/13954243537/job/39061393876?pr=15572